src/VisualStudio/Core/Def/Implementation/FindReferences/VisualStudioDefinitionsAndReferencesFactory.cs

```diff
@@ -117,6 +117,7 @@ private class ExternalDefinitionItem : DefinitionItem
             public override Task<bool> CanNavigateToAsync(Workspace workspace, CancellationToken cancellationToken)
                 => SpecializedTasks.True;
 
+            [Obsolete]
```

Comment:
What is preventing deletion here?

Reply:
i will look into this.

Reply:
this subclasses hte DefinitionItem guy that i'm not sure is totally isolated from TS/F#.  I will figure out what the EA story is there and delete if i can in followup.